File: soultions/1_two_sum.py
# Given a list of numbers return the two numbers that add up to the target number.
class Solution(object):
    def twoSum(self, nums, target):

        x = 0
        end = False

        while x < len(nums):
            difference = target - nums[x]
            current = nums[x]

            y =0
            while y < len(nums):
                if nums[y] == difference and y != x:
                    end = True
                    if end == True:
                        break
                y = y + 1
            if end == True:
                break

            x = x + 1

        return [x, y]

File: solutions/1_two_sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int x = 0;
        bool end = false;
        int num_size = nums.size();
        int current;
        std::vector<int> vec_of_ans;
        
        while (!end)
        {
            for (int i = 0; i < num_size; i++)
            {
                current = nums[i];
                for (int j = i + 1; j < num_size; j++)
                    if ((current + nums[j]) == target)
                    {
                        end = true;
                        
                        vec_of_ans.push_back(i);
                        vec_of_ans.push_back(j);
                    }
            }
        }
        return vec_of_ans;
    }
};
